src/metabase/lib/expressions.cljc

```diff
@@ -0,0 +1,143 @@
+(ns metabase.lib.expressions
```

Comment:
I think this should be singular for consistency with `lib.schema.expression`

Reply:
good call. I've changed it